tests/unit/test_checkpointing.py

```diff
@@ -0,0 +1,93 @@
+from unittest.mock import MagicMock
+from unittest import TestCase
+from fenics import FunctionSpace, UnitSquareMesh, Expression, interpolate
+from fenicsprecice.solverstate import SolverState
+
+
+class TestCheckpointing(TestCase):
+    def test_solverstate_basic(self):
+        """
+        Check if correct values are read from the checkpoint, while the state of the object that is copied is not changed
+        """
+        time = 0.5
+        iteration = 1
```

Comment:
```suggestion
        n = 1
```
why not `n` to make it consistent with `n_cp`? Same for below.